jedeschule/spiders/sachsen_anhalt.py

```diff
@@ -4,24 +4,64 @@
 import scrapy
 from scrapy.shell import inspect_response
 
+import requests
+import sys
+from lxml import html
+
+
+try:
+    import HTMLParser # to parse content from HTML school-types
+    from urlparse import parse_qs 
+except ImportError: # for Python 3
+    from html.parser import HTMLParser
+    from urllib.parse import parse_qs 
+
 
 class SachsenAnhaltSpider(scrapy.Spider):
+    
     name = "sachsen-anhalt"
-    start_urls = ['https://www.bildung-lsa.de/ajax.php?m=getSSResult&q=&lk=-1&sf=-1&so=-1&timestamp=1480082277128/']
-
     detail_url = "https://www.bildung-lsa.de/ajax.php?m=getSSDetails&id={}&timestamp=1480082332787"
+    
+
+   # must return an iterable with the first Requests to crawl for this spider
+    def start_requests(self):
+        page = requests.get('https://www.bildung-lsa.de/schule.html')
+        tree = html.fromstring(page.content)
+        self.TYPES_DICT = {}
+
+        options = tree.xpath('//form[@name="SSFORM"]/select[@name="sf"]/option') #content + id
+        
+        self.TYPES_DICT = dict([(o.get('value'), o.text.replace("nur ", "")) for o in options if o.get('value') != '-1'])
```

Comment:
I also find it a little unconventional here to start the process with `requests` instead of just relying on scrapy.

I slightly refactored this into a version that I think follows our current coding style more closely. Maybe you can take a look if it still works as expected and let me know what you think about it?
https://github.com/Datenschule/jedeschule-scraper/blob/pr/jedeschule/spiders/sachsen_anhalt.py

I'm not very happy with the more complex syntax in the dict comprehension now but think that the overall code is more understandable.

Reply:
@Nictec would it be possible for you to try this? ☝️

Reply:
Hi, 
I spoke with @Nictec  and we'll have a look after this. He just needs some time cause he is busy now.